products/pdt/cd/overlays/staging/certificate.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: networking.gke.io/v1
+kind: ManagedCertificate
+metadata:
+  name: pdt
+spec:
+  domains:
+    - pdt.zilstg.dev
```

Comment:
Is this necessary if you don't have a `Service` or `Ingress`?

Reply:
Can I just remove this file then?

Reply:
I would think so yes. I don't think we have anything to serve at `pdt.zilstg.dev`.